src/BrowserStorage.fs

```diff
@@ -0,0 +1,28 @@
+module Fable.PowerPack.BrowserStorage
```

Comment:
If we change this line to `namespace Fable.PowerPack` and the module names to `module BrowserLocalStorage` and `module BrowserSessionStorage` we don't need to deprecate the current API.

Reply:
Done 👍